common/components/ui/Aux.tsx

```diff
@@ -0,0 +1,3 @@
+const Aux = ({ children }) => children;
+
+export default Aux;
```

Comment:
I don't see this used anywhere, and it looks like a noop.

Reply:
I was initially using it to return groups of elements without needlessly wrapping them in a div, guess I got rid of its usage later on